app/sessionStore.js

```diff
@@ -926,7 +926,8 @@ module.exports.defaultAppState = () => {
     },
     cache: {
       bookmarkLocation: undefined,
-      bookmarkOrder: {}
+      bookmarkOrder: {},
+      ledgerVideos: {}
```

Comment:
We need to add deep merge for state migration. Suggestion by @bsclifton https://github.com/KyleAMathews/deepmerge

Reply:
will add it after PR will be confirmed